index.js

```diff
@@ -1605,9 +1648,11 @@ module.exports = function (log, indexesPath) {
     push(
       push.values(Object.entries(indexes)),
       push.asyncMap(([indexName, index], cb) => {
-        if (coreIndexNames.includes(indexName)) return cb()
-
-        if (index.lazy) {
+        if (coreIndexNames.includes(indexName)) {
```

Comment:
right, reindex can now be done when things are removed, so this changes the core indexes. Before when it was only encrypted, these core indexes does not change.